libsolidity/ast/Types.h

```diff
@@ -208,11 +208,20 @@ class Type
 	virtual bool operator==(Type const& _other) const { return category() == _other.category(); }
 	virtual bool operator!=(Type const& _other) const { return !this->operator ==(_other); }
 
-	/// @returns number of bytes used by this type when encoded for CALL. If it is a dynamic type,
-	/// returns the size of the pointer (usually 32). Returns 0 if the type cannot be encoded
+	/// @returns number of bytes used by this type when encoded for CALL. Cannot be used for
+	/// dynamically encoded types. Returns 0 if the type cannot be encoded
```

Comment:
It's slightly unclear here whether "cannot be used for dynamically encoded types" means "cannot be encoded" thus "returns 0". Maybe explictly say it will throw/assert?

Reply:
I think that comment was against the version before the last commit, right?
Now it doesn't return zero anymore anyways, but always throws and should be explicit enough.